scripts/python/sitecustomize.py

```
@@ -0,0 +1,182 @@
+    subcommand: str, exit_code: int, stdout: str, stderr: str
+) -> str:
+    """Return a consistent failure message for cargo pre-flight commands."""
+
+    message = f"Pre-flight cargo {subcommand} failed with exit code {exit_code}"
+    detail = (stderr or stdout).strip()
+    if detail:
+        message = f"{message}: {detail}"
+    return message
+
+
+def _patched_run_cargo_preflight(
```

Ruff (0.14.3)

Missing return type annotation for private function `_patched_run_cargo_preflight`

Add return type annotation: `None`

(ANN202)